client/lib/analytics/hotjar.js

```diff
@@ -7,7 +7,8 @@ import debug from 'debug';
  * Internal dependencies
  */
 import config from 'config';
-import { mayWeTrackCurrentUserGdpr, doNotTrack, isPiiUrl } from './utils';
+import { mayWeTrackCurrentUserGdpr, isPiiUrl } from './utils';
+import { getDoNotTrack } from '@automattic/calypso-analytics';
```

Comment:
Looks like `getDoNotTrack` is the default export: https://github.com/Automattic/wp-calypso/pull/39001/files#diff-6dd963982aebbb28e8af6ba227530323R11